Package.swift

```diff
@@ -41,4 +35,15 @@ let package = Package(
         .process("airbnb.swiftformat"),
         .process("swiftlint.yml"),
       ]),
+
+    // We don't always use official SwiftFormat releases from the nicklockwood/SwiftFormat
+    // repo, so we host our own artifactbundle instead of downloading one.
+    .binaryTarget(
+      name: "SwiftFormat",
+      path: "resources/SwiftFormat.artifactbundle"),
```

Comment:
Can we host this on your SwiftFormat fork? It seems like we probably don't want to check this in since it's a 6 MB executable

Reply:
Sure, I can zip it and attach it to the release for this version

Reply:
Updated in https://github.com/airbnb/swift/pull/181/commits/bad3bcb1d47eb0cddd8aaf44b814a03dafd8b2b6